frontend/app/views/info/feedback.scala.html

```diff
@@ -6,27 +6,41 @@
 
     <main role="main" class="page-content">
 
-        @fragments.page.pageHeader("Feedback", Some("Help us improve your membership experience"))
+        <section class="form-header">
```

Comment:
another example of this can be found [here](https://github.com/guardian/membership-frontend/pull/55/files#diff-c322c0d7e984f78f8e49bfc21c3e4b6eR14). It feels with this example that a fragment doesn't really give us much here